chromium/popup.js

```diff
@@ -275,3 +276,12 @@ function setOption_(opt, value, callback) {
   details[opt] = value;
   sendMessage("set_option", details, callback);
 }
+
+function getTab(callback) {
+  let url = new URL(window.location.href);
+  if (url.searchParams.has('tabId')) {
+    var parentId = parseInt(url.searchParams.get('tabId'));
```

Comment:
`const parentId = Number(url.searchParams.get('tabId'));`

Reply:
I think changing `var` to `let` here makes sense. `const` isn't used anywhere else in this file so it might be confusing. 

I'm not sure why `Number` would be better.